habana/pytorch/training/docker/1.12/py3/synapseai1.6.0/Dockerfile.hpu

```diff
@@ -0,0 +1,90 @@
+FROM vault.habana.ai/gaudi-docker/1.6.0/ubuntu20.04/habanalabs/pytorch-installer-1.12.0:latest
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Python versions
+ARG PYTHON=python3.8
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3.8
+
+ENV PT_VERSION=1.12
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+    build-essential \
+    openssh-client \
+    openssh-server \
+    ca-certificates \
+    curl \
+    jq \
+    emacs \
+    git \
+    libtemplate-perl \
+    libssl1.1 \
+    openssl \
+    unzip \
+    wget \
+    vim \
+    zlib1g-dev \
+    # Install dependent library for OpenCV
+    libgtk2.0-dev \
+ && rm -rf /var/lib/apt/lists/* \
+ && apt-get clean
+
+
+ARG PT_TRAINING_URL=https://framework-binaries.s3.us-west-2.amazonaws.com/habana/1.6.0/torch-1.12.0a0%2Bgit0a6bad0-cp38-cp38-linux_x86_64.whl 
+
+RUN pip --no-cache-dir install --upgrade \
+    pip \
+    "setuptools==59.5.0"
+
+# The following section uninstalls torch and torchvision before installing the
+# custom versions from an S3 bucket.
+RUN pip install --no-cache-dir --user -U \
+    fastai==1.0.61 \
+    "pyyaml>=5.4,<5.5" \
+    "awscli<2" \
+    psutil \
+    Pillow \
+    scipy \
+    pybind11 \
+    mpi4py==3.0.3 \
+    cmake==3.18.2.post1 \
+    torchnet \
+    "cryptography>3.2" \
+    werkzeug \
+ && pip install --no-cache-dir --user -U boto3 --upgrade \
+ && pip uninstall -y torch \
+ && pip uninstall -y hb-torch \
+ && pip install --no-cache-dir --user -U ${PT_TRAINING_URL} \
+ && rm -rf ${PT_TRAINING_URL}
+
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/root/.local/lib
+
+#RUN ln -s $(which ${PYTHON}) /usr/local/bin/python \
```

Comment:
why is this commented?

Reply:
@bukeao ^^

Reply:
it prints: 
ln: failed to create symbolic link '/usr/bin/pip'
: File exists